src/pages/Profile/Top/Top.jsx

```diff
@@ -61,16 +65,23 @@ export default function Top({ count, setSettings, queues }) {
                     <img src={plus} style={{width:"19px", height:"19px", cursor:"pointer"}}/>
                 </RowContainer>
 
-                {
-                    queues.find(item => item.wrap === true)?.wrap ? (
-                        <RowContainer name="member" style={{alignItems:"center"}} onClick={setModalMembersSettings}>
-                            <ProfileAddQueue> ADD NEW MEMBER </ProfileAddQueue>
-                            <img src={plus} style={{width:"19px", height:"19px", cursor:"pointer"}}/>
-                        </RowContainer>
-                    ) : <></>
-                }
-            </ColumnContainer>
+                <RowContainer   
+                    name="member" 
+                    style={queues.find(item => item.wrap === true)?.wrap ? {visibility:"visible", alignItems:"center", marginBottom:"10px"} : {visibility:"hidden", alignItems:"center", marginBottom:"10px"}} 
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
                    style={queues.find((item) => item.wrap === true)?.wrap ? {visibility:"visible", alignItems:"center", marginBottom:"10px"} : {visibility:"hidden", alignItems:"center", marginBottom:"10px"}} 
```